Release 2.6.0 by request

jhunkeler opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Fix spacetelescope/drizzlepac#2097

[🐱](https://jira.stsci.edu/browse/HLA-1566)

jhunkeler committed:
Release 2.6.0 by request

jhunkeler commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.23.15.25.2

jhunkeler committed:
Merge branch 'main' into retro-260

jhunkeler committed:
Replace clobbered build script

jhunkeler committed:
Do not cross-compile on osx_arm64

jhunkeler commented:
@conda-forge-admin, please rerender

jhunkeler committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.24.11.58.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

jhunkeler committed:
You have to cross-compile

jhunkeler committed:
You have to cross-compile (all the way)

jhunkeler committed:
Disabled sizeof check

jhunkeler committed:
Apply sizeof patch

pllim reviewed (approved):
Shall we?

pllim commented:
OK so how long to wait before we can revert this (#37)? LoL
